pyproject.toml

```diff
@@ -39,29 +38,31 @@ dependencies = [
     "ruamel.yaml==0.18.*",
     "pygelf==0.4.*",
     "toml==0.10.*",
-    "python-dateutil==2.8.*",
-    "deepdiff==6.7.*",
+    "python-dateutil==2.9.*",
+    "deepdiff==7.0.*",
     "ordered-set==4.1.*",
     # Each time cryptography version is updated, need to regenerate scripts/ci/custom-hooks/hook-cryptography.py
-    "cryptography==42.0.4",
-    "paramiko==3.3.*",
-    "jsonschema==4.17.*",
-    "typing_extensions==4.7.*",
-    "pathvalidate==3.2.*"
+    "cryptography==42.0.7",
+    "paramiko==3.4.*",
+    "jsonschema==4.22.*",
+    "referencing==0.35.*",
+    "typing_extensions==4.11.*",
+    "pathvalidate==3.2.*",
+    "useful_types==0.2.*",
 ]
-requires-python = ">=3.7"
+requires-python = ">=3.8"
 
 [project.optional-dependencies]
 ansible = ["ansible==7.0.*"]
```

Comment:
Ansible is outdated for 1.5 years, and not updated without external request due to potentially high impact.

Reply:
what about 7.X updated version ? https://pypi.org/project/ansible/7.7.0/

Reply:
https://github.com/Netcracker/KubeMarine/pull/661/commits/76c7c04b4847ca92d0dfd20fdf00877aee7bb1b2